packages/dso-toolkit/src/components/anchor/anchor.args.ts

```diff
@@ -23,15 +24,15 @@ export const anchorArgTypes: ArgTypes<AnchorArgs> = {
       type: "select",
     },
   },
-  label: {
+  mode: {
+    options: ["download", "extern"],
```

Comment:
Moeten we hier ook `undefined` toevoegen als optie?

Reply:
Lijkt mij inderdaad handig

Reply:
👍